### PR TITLE
ROB: cope with missing /W entry (#1134)

### PR DESCRIPTION
**Pull request by pubpub-zz**
robustness issue identified in #1134

### CODE REVIEW

File: PyPDF2/_cmap.py [hunk 1]
```diff
@@ -284,7 +284,10 @@ def compute_space_width(
             w1[-1] = cast(float, ft1["/DW"])
         except Exception:
             w1[-1] = 1000.0
-        w = list(ft1["/W"])  # type: ignore
+        try:
+            w = list(ft1["/W"])  # type: ignore
+        except Exception:
+            w = []
```
Comment (MartinThoma) [inline PyPDF2/_cmap.py:290]: Would the following be equivalent?

```suggestion
        if "/W" in ft1:
            w = list(ft1["/W"])  # type: ignore
        else:
            w = []
```

Comment (pubpub-zz) [reply PyPDF2/_cmap.py:290]: @MartinThoma,
sound good. any reason to prefer (for my improvement 😊)

Comment (MartinThoma) [reply PyPDF2/_cmap.py:290]: However, I don't have strong feelings about this one. If you have a reason for your preference, I will accept & merge that.

**Comment by MartinThoma** [comment]
You're having a streak! I wish I could do something similar to the Unreal Turnament games for killing bugs - MEGA Kill - ULTRA Kill - ... :smile:

**Pull request merged**